tests/js/spec/components/events/interfaces/stacktrace.spec.tsx

```diff
@@ -27,10 +27,6 @@ function renderedComponent(
 }
 
 describe('StackTrace', function () {
```

Comment:
this should not be part of this PR, but I could not resist, pls lmk if that is fine 😄